__test__/screens/ViewDetails/ViewAnnouncementScreen/ViewAnnouncementScreen.test.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react"
+import { render } from "@testing-library/react-native"
+import ViewAnnoucementScreen from "../../../../screens/ViewDetails/ViewAnnouncementScreen/ViewAnnouncementScreen"
+import { Point } from "react-native-maps"
+import { Announcement } from "../../../../types/Annoucements"
+
+// Create a dummy Point
+const dummyPoint: Point = {
+    x: 40.7128,
+    y: -74.0060
+}
+
+// Create a dummy Announcement
+const dummyAnnouncement: Announcement = {
+    uid: "1",
+    title: "Community Art Exhibition",
+    location: "Downtown Community Center",
+    point: dummyPoint,
+    description: "Join us for an evening of art and conversation at our annual community art exhibition. Featuring works from local artists in a variety of mediums.",
+    interests: ["art", "community", "local culture"],
+    date: "2024-10-03T18:00:00Z"
+}
```

Comment:
these dummy announcements/events should be taken out once and for all. It is not possible anymore to do this. If you want to mock the fetching of announcements, create a ManageAnnouncement in the firebase folder and mock a function there please.

Reply:
I don't really see the point in mocking the database. If you look at the code the announcement is just passed as props to the screen. I would need to mock the database then fetch the announcement and them pass it at prop why not just pass it right away ? There is not database implicated in this screen so I'm only testing the UI.

Reply:
I don't really see the point in mocking the database. If you look at the code, the announcement is just passed as props to the screen. I would need to mock the database then fetch the announcement and them pass it at prop, why not just pass it right away ? There is not database implicated in this screen so I'm only testing the UI.